src/Autosuggest.js

```diff
@@ -183,6 +210,8 @@ export default class Autosuggest extends Component {
         }
 
         this.setState(newState);
+        this.onSuggestionUnfocused();
+        lastFocusedSuggestion = null;
```

Comment:
I feel like we shouldn't be setting `lastFocusedSuggestion` outside of `this.onSuggestionUnfocused()` and `this.onSuggestionFocused()` (this one doesn't exist at the moment). What do you think?